src/backend/distributed/commands/common.c

```diff
@@ -61,22 +61,24 @@ PostprocessCreateDistributedObjectFromCatalogStmt(Node *stmt, const char *queryS
 		return NIL;
 	}
 
-	ObjectAddress address = GetObjectAddressFromParseTree(stmt, false);
+	List *addresses = GetObjectAddressFromParseTree(stmt, false);
+	Assert(list_length(addresses) == 1);
```

Comment:
I think for the places where we have `Assert` we need 1 line comment:

```
/*  the code-path only supports a single object */
Assert(list_length(addresses) == 1);
```

or such